src/Http/Http.Abstractions/src/HttpMethods.cs

```diff
@@ -25,46 +28,82 @@ public static class HttpMethods
         public static readonly string Put = "PUT";
         public static readonly string Trace = "TRACE";
 
+        /// <summary>
+        /// Verifies the HTTP request method is CONNECT. 
+        /// <summary/>
+        /// <param name="method">HTTP request method.</param>
```

Comment:
Can you also add a `<returns>` element (to really cover our bases). I suggest (again based on [other existing docs](https://docs.microsoft.com/dotnet/api/system.text.rune.isvalid?view=netcore-3.0#System_Text_Rune_IsValid_System_Int32_))

```
/// <returns><see langword="true" /> if method is CONNECT; otherwise, <see langword="false" />.
```

(Updated for each individual method)